partials/docs-sidebar.hbs

```diff
@@ -14,104 +17,168 @@
+      /><h2 class="text-truffle">TRUFFLE TEAMS</h2>
     </a>
 
-    <div class="collapse teams {{showDocsCategory path "teams"}}" id="teamsDocs" data-parent="#docsSidebar">
+    <div
+      class="collapse teams {{showDocsCategory path "teams"}}"
+      id="teamsDocs"
+      data-parent="#docsSidebar"
+    >
       <ul>
         <a href="/docs/teams/overview"><li>Overview</li></a>
         <a href="/docs/teams/quickstart"><li>Quickstart</li></a>
       </ul>
       {{#each docs.teams as |categoryDocs category|}}
-        <h3>{{ category }}</h3>
+        <h3>{{category}}</h3>
         <ul>
           {{#each categoryDocs}}
-            <a href="/docs/teams/{{makeSlug category}}/{{makeSlug this}}"><li>{{ this }}</li></a>
+            <a href="/docs/teams/{{makeSlug category}}/{{makeSlug this}}"><li
+              >{{this}}</li></a>
           {{/each}}
         </ul>
       {{/each}}
     </div>
 
-    <a class="toggle-truffle suite-link" href="#truffleDocs" data-toggle="collapse">
-      <img class="suite-logo" src="/img/truffle-logomark.svg" alt="Truffle Logo" /><h2 class="text-truffle"><span class="narrow">T</span>RUFFLE</h2>
+    <a
+      class="toggle-truffle suite-link"
+      href="#truffleDocs"
+      data-toggle="collapse"
+    >
+      <img
+        class="suite-logo"
+        src="/img/truffle-logomark.svg"
+        alt="Truffle Logo"
+      /><h2 class="text-truffle"><span class="narrow">T</span>RUFFLE</h2>
     </a>
 
-    <div class="collapse truffle {{showDocsCategory path "truffle"}}" id="truffleDocs" data-parent="#docsSidebar">
+    <div
+      class="collapse truffle {{showDocsCategory path "truffle"}}"
+      id="truffleDocs"
+      data-parent="#docsSidebar"
+    >
       <ul>
         <a href="/docs/truffle/overview"><li>Overview</li></a>
         <a href="/docs/truffle/quickstart"><li>Quickstart</li></a>
       </ul>
       {{#each docs.truffle as |categoryDocs category|}}
-        <h3>{{ category }}</h3>
+        <h3>{{category}}</h3>
         <ul>
           {{#each categoryDocs}}
-            <a href="/docs/truffle/{{makeSlug category}}/{{makeSlug this}}"><li>{{ this }}</li></a>
+            <a href="/docs/truffle/{{makeSlug category}}/{{makeSlug this}}"><li
+              >{{this}}</li></a>
           {{/each}}
         </ul>
       {{/each}}
       <h3>Guides</h3>
       <ul>
-        <a href="/guides/debugging-a-smart-contract"><li>Debugging a smart contract</li></a>
-        <a href="/guides/debugger-variable-inspection"><li>Variable Inspection: Going Deeper with the Truffle Solidity Debugger</li></a>
-        <a href="/guides/using-infura-custom-provider"><li>Using Infura (or a custom provider)</li></a>
-        <a href="/guides/deploying-to-the-live-network"><li>Deploying to the Live Network</li></a>
-        <a href="/guides/creating-a-cli-with-truffle-3"><li>Creating an Ethereum-enabled command line tool with Truffle 3.0</li></a>
+        <a href="/guides/debugging-an-example-smart-contract"><li>Debugging a
```

Comment:
Renamed^

Reply:
Looks like you didn't change the name in the text, or am I missing something?

Reply:
```suggestion
        <a href="/guides/debugging-an-example-smart-contract"><li>Debugging an example
```

Reply:
Good catch, thanks!